app/assets/javascripts/components/timeline/timeline.jsx

```diff
@@ -208,6 +208,8 @@ const Timeline = createReactClass({
     // week meetings, which indicates a blackout week. Then insert the week itself.
     // The index 'i' represents the zero-index week number; both empty and non-empty
     // weeks are included in this numbering scheme.
```

Comment:
These block of comments probably needs to be updated?

Reply:
Actually, it probably needs to be moved to the selector.

Reply:
Okay. Will correct it right way.